### PR TITLE
Feature/hex img component

### PR DESCRIPTION
**Pull request by kumardennis**

### CODE REVIEW

**Review by edanchenkov — COMMENTED** [review]
@kumardennis Could you add a comprehensive example to storybooks? E.g. how this component can look like in different context with different props/states. You can check here https://github.com/PlatformOfTrust/react-elements-of-trust/pull/37/files

File: stories/index.js [hunk 1]
```diff
@@ -16,6 +17,15 @@ storiesOf('Button', module)
...
     .add('default', () => <Button text="Hello World!" />);
 
+storiesOf('Image', module).addParameters({
+    backgrounds: [
+        { name: 'light', value: '#fffef0' },
+        { name: 'dark', value: '#1e3b47' },
+        { name: 'purple', value: '#7955ff', default: true }
+    ],
+}).add('default', () => (
+    <Image src="https://i.pinimg.com/originals/aa/49/54/aa4954451f16847cb657e68e5f46538a.jpg"/>
```
Comment (edanchenkov) [inline stories/index.js:38]: Maybe add local image instead?

File: stories/index.js [hunk 2]
```diff
@@ -24,4 +25,29 @@ storiesOf('Image', module).addParameters({
...
+            { name: 'light', value: '#fffef0' },
+            { name: 'dark', value: '#1e3b47' },
+            { name: 'purple', value: '#7955ff', default: true }
+        ]
+    })
+    .add('default', () => (
+        <div>
+            <HexImg
+                url={
+                    'https://penasultra.com/wp-content/plugins/webp-express/test/test-pattern-tv.jpg'
```
Comment (edanchenkov) [inline stories/index.js]: Maybe add local image instead?

File: stories/index.js [hunk 3]
```diff
@@ -24,4 +25,29 @@ storiesOf('Image', module).addParameters({
...
+                }
+                scale={1}
+            />
+
+            <HexImg
+                url={
+                    'https://penasultra.com/wp-content/plugins/webp-express/test/test-pattern-tv.jpg'
```
Comment (edanchenkov) [inline stories/index.js]: Maybe add local image instead?

**Review by edanchenkov — CHANGES_REQUESTED** [review]

File: stories/index.js [hunk 4]
```diff
@@ -38,14 +38,14 @@ storiesOf('HexImg', module)
         <div>
             <HexImg
                 url={
-                    'https://penasultra.com/wp-content/plugins/webp-express/test/test-pattern-tv.jpg'
+                    require('./../src/Components/Image/test.jpg')
```
Comment (edanchenkov) [inline stories/index.js]: This image should be in storybook folder `stories`. Components should not contain any test files.

File: src/Components/HexImage/styles.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
+.hexagon {
+    position: relative;
+    width: 300px;
+    height: 173.21px;
+    margin: 86.6px 0;
+    background-position: center;
+    background-size: auto 346.4102px;
+}
+
+.hexTop,
```
Comment (victorv-sbp) [inline src/Components/HexImage/styles.scss:10]: You should use hyphen instead of camel case for css class names.

**Review by victorv-sbp — APPROVED** [review]
Just a quick note here: generally, it's not good practice to merge a branch into another if the merged branch is not approved or considered stable.

**Pull request merged**